### PR TITLE
Add new benchmark with to_a

### PR DESCRIPTION
**Pull request by klgilbert**
Added additional benchmark for scope + records.

### CODE REVIEW

**Comment by klgilbert** [comment]
I updated the benchmark with an unique index on email. @SamSaffron is that what you were looking for?

File: rails/benchmarks/bm_activerecord_scope_to_a.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+ActiveRecord::Base.establish_connection(ENV['DATABASE_URL'])
+
+ActiveRecord::Migration.verbose = false
+
+ActiveRecord::Schema.define do
+  create_table :users, force: true do |t|
+    t.string :name, :email
+    t.timestamps null: false
+    t.index(:email, unique: true)
```
Comment (tgxworld) [inline rails/benchmarks/bm_activerecord_scope_to_a.rb]: Just a nitpick here. Lets leave out the parenthesis for consistency with the lines above.

**Comment by SamSaffron** [comment]
Yeah looks good

Minor change like foobar00% though I am pretty sure the limit clause caps
the cost anyway

**Comment by klgilbert** [comment]
Updated it

**Comment by SamSaffron** [comment]
did you confirm it returns 10 results in an array on local? 

merging it ... lets get it backfilled asap, I want to see the results.

**Comment by klgilbert** [comment]
Yeah it returned an array of 10 results.

**Comment by tgxworld** [comment]
@SamSaffron @klgilbert The benchmark is up.

**Pull request merged**